src/lib/slide-toggle/slide-toggle.ts

```diff
@@ -83,13 +83,13 @@ export class MdSlideToggle implements AfterContentInit, ControlValueAccessor {
   @Input() id: string = this._uniqueId;
 
   /** Used to specify the tabIndex value for the underlying input element. */
-  @Input() tabIndex: number = 0;
+  @Input() tabindex: number = 0;
```

Comment:
But if we change the `@Input()` to `tabIndex` then using `tabindex` as a normal HTML attribute won't work anymore.

I think we should do the following (to cover both cases) 
```ts
@Input('tabindex') tabIndex
```

> Also we currently have `tabindex` for the deprecated input right now.